<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">

 <class>ConsInputDialog</class>

 <widget class="QDialog" name="ConsInputDialog">

  <property name="geometry">

   <rect>

    <x>0</x>

    <y>0</y>

    <width>994</width>

    <height>635</height>

   </rect>

  </property>

  <property name="minimumSize">

   <size>

    <width>983</width>

    <height>635</height>

   </size>

  </property>

  <property name="windowTitle">

   <string>Constraints Input</string>

  </property>

  <property name="windowIcon">

   <iconset resource="backgroundsrc.qrc">

    <normaloff>:/images/graylogo.png</normaloff>:/images/graylogo.png</iconset>

  </property>

  <property name="styleSheet">

   <string notr="true">#ConsInputDialog{

background: #c3c3c3;

border-image:url(:/images/finalBackground.jpg)

}



QSpinBox, QTextEdit, QDateTimeEdit, QPushButton, QLabel{

font-family: &quot;Roboto Black&quot;;

font-size: 42px;

}



QPushButton{

background-color:white;

padding: 10px;

}



QPushButton:hover{

background-color: #eb3f79;

color: white;

}



QRadioButton::indicator::unchecked{

border: 3px solid;

border-color: white;

border-radius: 6px;

background-color: black;

}



QRadioButton::indicator::checked{

border: 3px solid;

border-color: black;

border-radius: 6px;

background-color: #eb3f79

}







</string>

  </property>

  <widget class="QWidget" name="Content" native="true">

   <property name="geometry">

    <rect>

     <x>20</x>

     <y>170</y>

     <width>971</width>

     <height>441</height>

    </rect>

   </property>

   <widget class="QWidget" name="formLayoutWidget">

    <property name="geometry">

     <rect>

      <x>90</x>

      <y>40</y>

      <width>821</width>

      <height>311</height>

     </rect>

    </property>

    <layout class="QFormLayout" name="formLayout">

     <item row="1" column="1">

      <widget class="QTextEdit" name="input_name"/>

     </item>

     <item row="2" column="1">

      <widget class="QDateTimeEdit" name="input_startDate">

       <property name="font">

        <font>

         <family>Roboto Black</family>

         <pointsize>-1</pointsize>

        </font>

       </property>

       <property name="alignment">

        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>

       </property>

       <property name="dateTime">

        <datetime>

         <hour>0</hour>

         <minute>0</minute>

         <second>0</second>

         <year>2020</year>

         <month>11</month>

         <day>1</day>

        </datetime>

       </property>

       <property name="minimumDateTime">

        <datetime>

         <hour>0</hour>

         <minute>0</minute>

         <second>0</second>

         <year>2020</year>

         <month>11</month>

         <day>1</day>

        </datetime>

       </property>

      </widget>

     </item>

     <item row="4" column="0">

      <widget class="QLabel" name="label_minRequired">

       <property name="font">

        <font>

         <family>Roboto Black</family>

         <pointsize>-1</pointsize>

         <weight>75</weight>

         <bold>true</bold>

        </font>

       </property>

       <property name="text">

        <string>Time Needed       (1-1440 Mins)</string>

       </property>

       <property name="wordWrap">

        <bool>true</bool>

       </property>

      </widget>

     </item>

     <item row="4" column="1">

      <widget class="QSpinBox" name="input_minRequired">

       <property name="font">

        <font>

         <family>Roboto Black</family>

         <pointsize>-1</pointsize>

        </font>

       </property>

       <property name="alignment">

        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>

       </property>

       <property name="minimum">

        <number>1</number>

       </property>

       <property name="maximum">

        <number>1440</number>

       </property>

       <property name="singleStep">

        <number>1</number>

       </property>

       <property name="value">

        <number>1</number>

       </property>

      </widget>

     </item>

     <item row="1" column="0">

      <widget class="QLabel" name="label_name">

       <property name="minimumSize">

        <size>

         <width>350</width>

         <height>0</height>

        </size>

       </property>

       <property name="font">

        <font>

         <family>Roboto Black</family>

         <pointsize>-1</pointsize>

         <weight>75</weight>

         <bold>true</bold>

        </font>

       </property>

       <property name="text">

        <string>Constraint Name</string>

       </property>

      </widget>

     </item>

     <item row="2" column="0">

      <widget class="QLabel" name="label_startDate">

       <property name="font">

        <font>

         <family>Roboto Black</family>

         <pointsize>-1</pointsize>

         <weight>75</weight>

         <bold>true</bold>

        </font>

       </property>

       <property name="text">

        <string>Start Date</string>

       </property>

      </widget>

     </item>

    </layout>

   </widget>

   <widget class="QDialogButtonBox" name="buttonBox">

    <property name="geometry">

     <rect>

      <x>610</x>

      <y>350</y>

      <width>300</width>

      <height>61</height>

     </rect>

    </property>

    <property name="minimumSize">

     <size>

      <width>300</width>

      <height>61</height>

     </size>

    </property>

    <property name="font">

     <font>

      <pointsize>17</pointsize>

     </font>

    </property>

    <property name="standardButtons">

     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>

    </property>

   </widget>

  </widget>

  <widget class="QWidget" name="topPanel" native="true">

   <property name="geometry">

    <rect>

     <x>-30</x>

     <y>-10</y>

     <width>1029</width>

     <height>131</height>

    </rect>

   </property>

   <property name="minimumSize">

    <size>

     <width>1029</width>

     <height>131</height>

    </size>

   </property>

   <property name="styleSheet">

    <string notr="true">#topPanel{

border-image:url(:/images/mainHeader.png)

}</string>

   </property>

  </widget>

  <widget class="QLabel" name="Title">

   <property name="geometry">

    <rect>

     <x>280</x>

     <y>140</y>

     <width>521</width>

     <height>59</height>

    </rect>

   </property>

   <property name="styleSheet">

    <string notr="true">QLabel{

font-weight: bold;

font-size: 52px;

}



</string>

   </property>

   <property name="text">

    <string>Constraints Input</string>

   </property>

  </widget>

 </widget>

 <resources>

  <include location="backgroundsrc.qrc"/>

 </resources>

 <connections/>

</ui>

